pymc3/stats.py

```diff
@@ -1012,7 +1018,8 @@ def _groupby_leading_idxs(shape):
 
 
 def bfmi(trace):
-    R"""Calculate the estimated Bayesian fraction of missing information (BFMI).
```

Comment:
Unnecessary change (same the one below).

Reply:
Thanks @junpenglao ! I'm not sure what happened with my upstream master merging, but I'll make sure to look into that before starting any further PR.